docs/install/redhat.rst

```diff
@@ -28,11 +28,27 @@ from the CrateDB package repository::
     # Import the public GPG key for verifying the package signatures.
     sudo rpm --import https://cdn.crate.io/downloads/yum/RPM-GPG-KEY-crate
```

Comment:
Hopefully, this command will drop the key at `/etc/pki/rpm-gpg/RPM-GPG-KEY-crate`, to match the repository configuration below? Actually, I did not validate the changes yet. Maybe someone of you can spare a few minutes?

Reply:
It was not stored under that location on Amazon Linux 2023, but everything still worked fine, with no warnings or errors. Potentially yum automatically knew where keys were stored and didn't attempt to check the manually specified path? Not sure, other repositories are using that path also:

```
[root@ip-172-31-26-116 ec2-user]# cat /etc/yum.repos.d/amazonlinux.repo 
[amazonlinux]
name=Amazon Linux 2023 repository
mirrorlist=https://al2023-repos-$awsregion-de612dc2.s3.dualstack.$awsregion.$awsdomain/core/mirrors/$releasever/$basearch/mirror.list
priority=10
enabled=1
repo_gpgcheck=0
type=rpm
gpgcheck=1
gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-amazon-linux-2023

[amazonlinux-source]
name=Amazon Linux 2023 repository - Source packages
mirrorlist=https://al2023-repos-$awsregion-de612dc2.s3.dualstack.$awsregion.$awsdomain/core/mirrors/$releasever/SRPMS/mirror.list
enabled=0
repo_gpgcheck=0
type=rpm
gpgcheck=1
gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-amazon-linux-2023
metadata_expire=6h

[amazonlinux-debuginfo]
name=Amazon Linux 2023 repository - Debug
mirrorlist=https://al2023-repos-$awsregion-de612dc2.s3.dualstack.$awsregion.$awsdomain/core/mirrors/$releasever/debuginfo/$basearch/mirror.list
enabled=0
repo_gpgcheck=0
type=rpm
gpgcheck=1
gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-amazon-linux-2023
metadata_expire=6h
[root@ip-172-31-26-116 ec2-user]# ls /etc/pki/rpm-gpg/
RPM-GPG-KEY-amazon-linux-2022  RPM-GPG-KEY-amazon-linux-2023
[root@ip-172-31-26-116 ec2-user]# rpm --import https://cdn.crate.io/downloads/yum/RPM-GPG-KEY-crate 
[root@ip-172-31-26-116 ec2-user]# ls /etc/pki/rpm-gpg/
RPM-GPG-KEY-amazon-linux-2022  RPM-GPG-KEY-amazon-linux-2023
```

Reply:
> The `gpgkey` setting also supports a URL. So we can skip the manual `rpm --import` and just set `gpgkey=https://cdn.crate.io/downloads/yum/RPM-GPG-KEY-crate`.

That sounds sweet. If there are no other objections, I will change it like this.

Reply:
Just to share my observations, probing the installation on CentOS 9 Stream, and AlmaLinux 9.

```
racker --verbose run -it --rm almalinux:9 /bin/bash
racker --verbose run -it --rm quay.io/centos/centos:stream9 /bin/bash
```

I missed to check on AlmaLinux, but on CentOS, the `gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-crate` setting does not have any kind of relevance any longer. When using `rpm --import` to import a public GPG key, it will no longer be located into `/etc/pki/rpm-gpg` at all, apparently.

Reply:
The patch has been amended to do it like advised. It seems to work well. Thanks.